tests/test_avro_producer.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+#
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+#
+# derived from https://github.com/verisign/python-confluent-schemaregistry.git
+#
+import sys
+
+from confluent_kafka.avro.serializer import util
+
+if sys.version_info[0] < 3:
+    import unittest
+else:
+    import unittest2 as unittest
+from confluent_kafka.avro.avro_producer import AvroProducer
+
+
+class TestAvroProducer(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def test_instantiation(self):
+        obj = AvroProducer(None, 'http://127.0.0.1:9002')
```

Comment:
If possible, it would be better to not depend on a specific port. This is a really common way to generate confusion when developers have something else running on that port or to cause random failures of tests on shared infrastructure like Jenkins. We had lots of these problems in Kafka until I converted all the places we bind to ports to use `0` (to select a random available port) and then constructing URLs to match whatever port ends up being bound.